src/System.Net.Security/src/System/Net/Security/SslStream.cs

```diff
@@ -525,21 +525,8 @@ public override void EndWrite(IAsyncResult asyncResult)
         // Returns:
         // 
         //     A Task<int> representing the read.
-        public override Task<int> ReadAsync(byte[] buffer, int offset, int size, CancellationToken cancellationToken)
-        {
-            if (cancellationToken.IsCancellationRequested)
-            {
-                return Task.FromCanceled<int>(cancellationToken);
-            }
-
-            return Task.Factory.FromAsync(
-                (bufferArg, offsetArg, sizeArg, callback, state) => ((SslStream)state).BeginRead(bufferArg, offsetArg, sizeArg, callback, state),
-                iar => ((SslStream)iar.AsyncState).EndRead(iar),
-                buffer,
-                offset,
-                size,
-                this);
-        }
+        public override Task<int> ReadAsync(byte[] buffer, int offset, int size, CancellationToken cancellationToken) =>
```

Comment:
> Are you asking why I didn't just delete the method entirely

Yes.

> a) overrides can affect the contract

Right, I remember that code.

Might be worth adding a comment in the code so people don't try to "fix" it.